### DISCUSSION TITLE
Bing ai chat unrestricted?

### DISCUSSION THREAD

**Discussion by PigletsInHay** [juzeon/SydneyQt#113] [Q&A]
Hello, I just started using Bing AI for generating art but it's so difficult due to the forbidden words that seem to be a whole lot.
It just never tells what words are forbidden to use and it's a guessing game most of the time until you get banned out of the blue...

Is it possible to use something here on the site for making unrestricted art or is it only for the chat version? 

Thankful if someone could explain this to me.

**Comment by HarshwardhanPatil07** [comment]
Hey @PigletsInHay 
To generate unrestricted art, you might want to explore alternative AI art generation tools or platforms. Different AI art generators have varying policies and guidelines. Some may offer more freedom in generating art without as many restrictions.

If you're specifically looking for an unrestricted experience, you may need to look for AI art generators that explicitly advertise this feature or check for updates from Bing AI to see if they have introduced new options or settings that allow for more creative freedom.

Keep in mind that AI art generation is still a relatively new field, and platforms and technologies are evolving. It's important to use these tools responsibly and respect any content guidelines or terms of service set by the platform you choose to use. Additionally, always be aware of any potential legal and ethical implications associated with the art you create using AI tools.
